_source/_posts/2021-09-08-sonar-qube-dotnet.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: blog_post
+title: "How to Write Cleaner, Safer Code with SonarQube, Docker and .NET Core"
+Author: [pawel-szydziak]
+by: contractor
+communities: [.net]
+description: "This tutorial demonstrates how to use SonarQube with .NET Core. "
+tags: [sonarqube, csharp-9, dotnet, dotnetcore, csharp, aspnet, aspnetcore, dotnet5]
+tweets:
+- "Learn all you need to know about #sonarqube + #dotnetcore in this new tutorial ->"
+- "Get up to speed on using #sonarqube and #dotnet in this post"
+- "Wondering how #sonarqube works with @oktadev in #dotnet? We've got you covered!"
```

Comment:
```suggestion
- "Wondering how #sonarqube works with @oktadev in #dotnet? We've got you covered!"
```

Reply:
I don't understand the change here. Are we removing the tweet?